.github/workflows/server_cd_release.yml

```
@@ -51,37 +53,26 @@ jobs:
           cd backend
           ./gradlew clean build -x test
 
-      - id: 'auth'
-        uses: 'google-github-actions/auth@v2'
+      # Docker 인증
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)